spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -1331,6 +1331,44 @@ The connector is responsible for the acknowledgment (positive or negative) of th
 * An outgoing connector must acknowledge the incoming `org.eclipse.microprofile.reactive.messaging.Message` once it has successfully dispatched the message.
 * An outgoing connector must acknowledge negatively the incoming `org.eclipse.microprofile.reactive.messaging.Message` if it cannot be dispatched.
 
+== Health
+
+When MicroProfile Reactive Messaging is used in an environment where MicroProfile Health is enabled, implicit readiness
+and liveness checks `messaging` are produced.
```

Comment:
`messaging` might not be explicit enough, or conflict with user checks. What about `MicroProfile Reactive Messaging`?

Reply:
You are right, I just can't find any definitive convention for naming health checks, maybe `mp-reactive-messaging` would make it more machine readable?